ui/src/app/job-list/table/table.component.ts

```diff
@@ -28,37 +33,40 @@ import {environment} from '../../../environments/environment';
   templateUrl: './table.component.html',
   styleUrls: ['./table.component.css'],
 })
+@Injectable()
 export class JobsTableComponent implements OnInit {
   @Input() dataSource: DataSource<QueryJobsResult>;
   @Output() onJobsChanged: EventEmitter<QueryJobsResult[]> = new EventEmitter();
 
   private mouseoverJob: QueryJobsResult;
 
-  public additionalColumns: string[] = [];
+  public displayFields: DisplayField[];
   public selection = new SelectionModel<QueryJobsResult>(/* allowMultiSelect */ true, []);
   public jobs: QueryJobsResult[] = [];
 
   // TODO(alanhwang): Allow these columns to be configured by the user
-  displayedColumns = primaryColumns.slice();
+  displayedColumns: string[] = ["Job", "Details"];
```

Comment:
what is "Details"?

Reply:
It's the Job details dropdown column [here](https://github.com/DataBiosphere/job-manager/pull/222/files/f3a063f766bd8dd9b94803e647f172242d0dbcb9#diff-906b98c1103b300e8cf6d2c5eb1a56edR24).